internal/environment/environment.go

```diff
@@ -24,6 +24,12 @@ var (
 	// hold Clever's lambda artifacts. There are 4 total – one for each
 	// region. The naming scheme is '<prefix>-<region>'
 	LambdaArtifactBucketPrefix = envMustString("LAMBDA_AWS_BUCKET", true)
+	// PreviousPipelineCompare is the git commit range to run change
+	// detection commands against when running for the primary branch.
+	PreviousPipelineCompare = envMustString("PREVIOUS_PIPELINE_COMPARE", false)
```

Comment:
With this, do the instructions for running locally need to be updated?

Reply:
I've got another ticket for this to take care of next: https://clever.atlassian.net/browse/INFRANG-6709